view/frontend/templates/css/boltcss.phtml

```diff
@@ -20,27 +20,6 @@
  *
  * @var $block \Bolt\Boltpay\Block\Js
  */
-if ($block->shouldDisableBoltCheckout()) return;
+if ($block->shouldDisableBoltCheckout()) ;
 ?>
-<style>
```

Comment:
You need the <style> tag to inject CSS into. Just remove the hardcoded CSS, leave the rest as is, that's it.